docs/website/docs/index.md

```diff
@@ -0,0 +1,22 @@
+# IREE
+
+** * UNDER CONSTRUCTION ** *
```

Comment:
Whoops, missed my latest commit :P will cherry-pick and update this PR. Standby...